.github/workflows/unit-tests-gh-hosted.yml

```diff
@@ -65,9 +67,11 @@ jobs:
 
       - name: Install Required Python Dependencies
         run: |
+            sed -i '/problem/ s/# //' common/lib/xmodule/setup.py
```

Comment:
this line uncomments line 27 in common/lib/xmodule/setup.py. we do this in order to allow a problem entry point to exist when the unit tests run. in production, our medality plugin exposes an entry point for `problem` which points to a subclass of `xmodule.capa_module.ProblemBlock`.